apps/game_backend/lib/game_backend/users.ex

```diff
@@ -77,7 +77,11 @@ defmodule GameBackend.Users do
     q =
       from(u in User,
         where: u.id == ^id and u.game_id == ^game_id,
-        preload: [units: [:character, :items], currencies: :currency]
+        left_join: unit in Unit,
```

Comment:
Why `left_join` and not `join`?

Reply:
you're right! this was the first iteration of this query and then it got changed to `join` since there were some users without units. Changed!